src/frontend/components/Chat/SharedChatComponents/ChatMessageMetadatas/index.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { ChatMessageMetadatas } from '.';
+import { wrapInIntlProvider } from '../../../../utils/tests/intl';
```

Comment:
ditto path

Reply:
fix in [30417701d10c51da927b5c824ee5a16a5259c4a5](https://github.com/openfun/marsha/pull/1247/commits/30417701d10c51da927b5c824ee5a16a5259c4a5)